node_tree.py

```diff
@@ -558,17 +505,6 @@ def error(self, msg, *args, **kwargs):
     def exception(self, msg, *args, **kwargs):
         self.getLogger().exception(msg, *args, **kwargs)
 
-    def set_implicit_conversions(self, input_socket_name, policy):
-        """
-        Set implicit conversion policy to be used by default for specified input socket.
-        This policy will be used by default by subsequent .sv_get() calls to this socket.
-        Policy can be passed as direct reference to the class, or as a class name.
-        """
-        if isinstance(policy, str):
-            policy = getattr(sverchok.core.socket_conversions, policy)
-        #self.debug("Set default conversion policy for socket %s to %s", input_socket_name, policy)
-        self._implicit_conversion_policy[input_socket_name] = policy
-
```

Comment:
"implicit convertions" are actually used; maybe "set_implicit_conversions" is not used at the moment, but since the whole thing is used, it can appear it will be used later; I'd suggest to leave the method for now.

Reply:
It have gone two years since this PR: #2024 and only policy is used now is DefaultImplicitConversionPolicy. Probably we could simplify below method in this way:
```diff
class SvSocketCommon:
    def convert_data(self, source_data, implicit_conversions=None):
        if not self.needs_data_conversion():
            return source_data
        else:
-            policy = self.node.get_implicit_conversions(self.name, implicit_conversions)
-            self.node.debug(f"Trying to convert data for input socket {self.name} by {policy}")
-            return policy.convert(self, source_data)
+            return DefaultImplicitConversionPolicy.convert(self, cource_data)
```

And simplify node class as well:

```diff
class SverchCustomTreeNode:
-    _implicit_conversion_policy = dict()
-    def set_implicit_conversions(self, input_socket_name, policy): ...
-    def get_implicit_conversions(self, input_socket_name, override=None): ...
```

It won't impact on anything but will make code easier. 

And I hardly can imagine how different nodes can use different conversion policy. It will be difficult to predict what output to expect from different nodes.

Reply:
At least Lenient policy is used somewhere, as far as I remember... In SvStringsSocket?...

Reply:
Nowhere
![2020-08-28_19-13-14](https://user-images.githubusercontent.com/28003269/91583630-8d9a7c00-e962-11ea-9a28-f7875ed27f81.png)
But I'm not going to delete this class at least not in this PR.